flict/var/relicense.json

```diff
@@ -8,16 +8,16 @@
         {
             "spdx": "GPL-2.0-or-later",
             "later": [
-                "GPL-3.0-only",
-                "GPL-2.0-only"
+                "GPL-3.0-or-later",
+                "GPL-2.0-or-later"
             ]
         },       
         {
             "spdx": "LGPL-2.1-or-later",
             "later": [
-                "GPL-2.0-only",
-                "LGPL-2.1-only",
-                "LGPL-3.0-only"
+                "GPL-2.0-or-later",
```

Comment:
That doesn't look right to me - lgpl can't turn into GPL (no matter what version)

Reply:
https://www.gnu.org/licenses/old-licenses/lgpl-2.0.html

_3. You may opt to apply the terms of the ordinary GNU General Public License instead of this License to a given copy of the Library. To do this, you must alter all the notices that refer to this License, so that they refer to the ordinary GNU General Public License, version 2, instead of to this License. (If a newer version than version 2 of the ordinary GNU General Public License has appeared, then you can specify that version instead if you wish.) Do not make any other change in these notices._

_Once this change is made in a given copy, it is irreversible for that copy, so the ordinary GNU General Public License applies to all subsequent copies and derivative works made from that copy._

_This option is useful when you wish to copy part of the code of the Library into a program that is not a library._

Reply:
I am not a lawyer, needless to say

Reply:
@silverhook any idea?